packages/react/src/components/SwatchColorPicker/SwatchColorPicker.types.ts

```diff
@@ -88,6 +88,11 @@ export interface ISwatchColorPickerProps extends React.RefAttributes<HTMLElement
    */
   onRenderColorCell?: IRenderFunction<IColorCellProps>;
 
+  /**
+   * Custom render function for button/cell content area
+   */
+  onRenderColorCellContent?: IRenderFunction<IColorCellProps>;
```

Comment:
I'm also curious why there is onRenderColorCellContent and onRenderCellContent

2 asks:
1. Write an example of how these new props would be used to customize the control (either in PR description, or by creating a new website example). I'm curious if you need to use both onRenders or just one.
2. explain the reasoning for the names being different. and make it obvious to a user which onRender they should be worried about.

Reply:
Will rename these to have the same name `onRenderColorCellContent`. 
These are on two different components, one on SwatchColorPicker, which calls `ColorPickerGridCell`, so users generally need to only pass one prop to SwatchColorPicker.

Reply:
Okay, that makes sense. I like them better as the same name. Makes it obvious that the parent is just passing this same named prop to all of its children. The approach we typically take in newer v9 is to use context to pass the values down, but with v8 it isn't required due to parent manually rendering the children.

Thanks for the demo, looks great!